src/main/java/org/bahmni/batch/form/ObservationProcessor.java

```diff
@@ -59,7 +59,14 @@ public List<Obs> process(Map<String,Object> obsRow) throws Exception {
 			allChildObsIds.add((Integer)obsRow.get("obs_id"));
 		}
 
-		List<Obs> obsRows = fetchAllLeafObs(allChildObsIds);
```

Comment:
Wouldn't it be better if we just change things in `fetchAllLeafObs` to return an empty List if the `allChildObsIds` passed in are null or an empty List?  Then leave the rest of this method alone?